factory-layout/src/main/resources/application.properties

```diff
@@ -5,3 +5,4 @@ spring.datasource.username = ${JavaWarriorsUser}
 spring.datasource.password = ${JavaWarriors}
```

Comment:
The properties look correct to have connection to the database. You MAY need to add another dependency later on... maybe.